### PR TITLE
MAUI-871321-Added theme keys for MAUI Shimmer, ProgressBar, Picker

### PR DESCRIPTION
**Pull request by Priyadharshini-Periyasamy**
### **Description**

- Added theme keys for below .NET MAUI controls

       -  SfShimmer

       -  SfLinearProgressbar

       -  SfCircularProgressbar

       -  SfPicker, SfDatePicker, SfTimePicker, SfDateTimePicker

### CODE REVIEW

File: MAUI/Themes/Keys.md [hunk 1]
```diff
@@ -107,3 +107,1293 @@ This page lists the keys for each control and the element to which it is mapped
...
+            <br/>
+        </td>
+        <tr>
+        <td>
+            SfLinearProgressBarTrackColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Trach color of the linear progress bar.
```
Comment (MuniappanSubramanian) [inline MAUI/Themes/Keys.md:268]: Trach - Check spelling

File: MAUI/Themes/Keys.md [hunk 2]
```diff
@@ -107,3 +107,1293 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+	</tr>
+    <tr>
+        <td>
+            SfLinearProgressBarBackground 
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Progress bar background of the linear progress bar.
```
Comment (MuniappanSubramanian) [inline MAUI/Themes/Keys.md:281]: **Progress bar** background of the **linear progress bar.**

Progress bar mentioned - continusouly

File: MAUI/Themes/Keys.md [hunk 3]
```diff
@@ -107,3 +107,1293 @@ This page lists the keys for each control and the element to which it is mapped
...
+        </td>
+    </tr>
+    <tr>
+        <td>
+            SfLinearProgressBarProgressColor 
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Progress bar color of the linear progress bar.
```
Comment (MuniappanSubramanian) [inline MAUI/Themes/Keys.md:293]: Progress bar color - Mentioned as progress bar color but API name is ProgressColor

File: MAUI/Themes/Keys.md [hunk 4]
```diff
@@ -107,3 +107,1293 @@ This page lists the keys for each control and the element to which it is mapped
...
+        </td>
+    </tr>
+    <tr>
+        <td>
+            SfLinearProgressBarSecondaryProgressColor 
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Progress bar secondary progress color of the linear progress bar.
```
Comment (MuniappanSubramanian) [inline MAUI/Themes/Keys.md:305]: Check "Progress bar" need to add. 

Check in circular pogressbar also

File: MAUI/Themes/Keys.md [hunk 5]
```diff
@@ -107,3 +107,1293 @@ This page lists the keys for each control and the element to which it is mapped
...
+        </td>
+        <td>
+            Selected text color of the picker.
+        <br/>
+        <br/>
+        </td>
+    </tr>
+    <tr>
+        <td>
+            SfPickerNormalTextColor
```
Comment (MuniappanSubramanian) [inline MAUI/Themes/Keys.md:658]: Screenshot 2024-03-12 at 1 26 52 PM
Check table alignment

File: MAUI/Themes/Keys.md [hunk 6]
```diff
@@ -107,3 +107,1293 @@ This page lists the keys for each control and the element to which it is mapped
...
+        </th>
+    </tr>
+    <tr>
+        <td rowspan="14">
+            SfDatePickerStyles  
+            <br/>
+            <br/>
+        </td>
+		<td>
+           SfDatePickerTheme
```
Comment (MuniappanSubramanian) [inline MAUI/Themes/Keys.md:694]: Screenshot 2024-03-12 at 1 27 51 PM

check table alignment

**Review by florenceliliana — COMMENTED** [review]
The content provided is fine.

**Review by MuniappanSubramanian — APPROVED** [review]
Changes are fine

**Pull request merged**